thirdeye/thirdeye-frontend/app/mocks/sampleEvents.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * Mock events for rca-poc
+ */
+export default [
+  {
+    "urn": "thirdeye:event:holiday:123456789",
+    "score": 0.05,
+    "label": "Halloween",
+    "type": "event",
+    "link": "http://google.com",
+    "relatedEntities": [
+    ],
+    "start": 1509580800000,
+    "end": 1509667200000,
+    "eventType": "holiday",
+    "details": "",
+    duration: 86400000,
+    relStart: 46800000,
+    relEnd: null,
+    relDuration: null,
+    isBaseline: false,
+    isFuture: true,
+    isOngoing: false,
+    displayStart: 1509476399999,
+    displayEnd: 1509479999999,
+    displayLabel: 'Halloween',
+    color: 'green',
+    displayColor: 'green',
+    displayScore: 0.175,
+    humanRelStart: '13 hours after',
+    humanDuration: 'ongoing',
+    "attributes": {
+      "country": ["ca", "ca"],
+      "region": ["north america"]
+    }
+  }, {
+    "urn": "thirdeye:event:deployment:123456789",
+    "score": 0.999,
+    "label": "my system",
+    "type": "event",
+    "link": "http://google.com",
+    "relatedEntities": [
+    ],
+    "start": 1509566414000,
+    "end": 1509652772000,
+    "eventType": "deployment",
+    "details": "",
+    duration: 744000,
+    relStart: -945000,
+    relEnd: -201000,
+    relDuration: 744000,
+    isBaseline: false,
+    isFuture: false,
+    isOngoing: false,
+    displayStart: 1509471855000,
+    displayEnd: 1509472599000,
+    displayLabel: 'helpcenter [prod-ltx1] (10:44 am)',
+    color: 'red',
+    displayColor: 'red',
+    displayScore: 0.025,
+    humanRelStart: '16 mins before',
+    humanDuration: '12 mins',
+    "attributes": {
+      "fabric": ['prod']
```

Comment:
where you trying to create a gcn event? looks like youre config file above is expecting : +    header: "gcn",
```
 +    inputs: [
 +      {
 +        label: "priority",
 +        type: "dropdown"
 +      }
 +    ]
```